src/extensions/lens-extension.ts

```diff
@@ -34,7 +33,6 @@ export class LensExtension implements ExtensionModel {
   @observable manifest: ExtensionManifest;
   @observable manifestPath: string;
   @observable isEnabled = false;
-  @observable.ref runtime: LensExtensionRuntimeEnv;
```

Comment:
Removed initially planned "runtime". Now everything could be served via webpack's externals into global Lens runtime.